README.md

```diff
@@ -16,6 +16,13 @@ See [Choosing Secure Passwords](https://www.schneier.com/blog/archives/2014/03/c
 This feature enables the HTTP header `X-XSS-Protection: 1; mode=block` to be sent on all requests which some web browsers intend as a way to automatically block suspected cross-site scripting attacks.
 Several web browsers (e.g., Firefox, Edge, and Chrome) do not support this header.
 
+## Require Overall/Read for Specific HTTP Headers
+
+This feature adds a permission check for `Overall/Read` in order to see HTTP headers that have been configured to be filtered.
```

Comment:
I suggest re-structuring this documentation a little. Especially this first line feels a little inverted. It might be better to lead with something more of a problem statement. Something more like one of the later lines. It would also be good to clarify what "to be filtered" means. Filtering in? Out?